src/Microsoft.NetCore.Analyzers/UnitTests/ImmutableCollections/DoNotCallToImmutableCollectionOnAnImmutableCollectionValueTests.Fixer.cs

```diff
@@ -125,10 +108,10 @@ class C
 {{
     public void M(IEnumerable<KeyValuePair<int, int>> p1, List<KeyValuePair<int, int>> p2, {collectionName}<int, int> p3)
     {{
-        var a = p1.To{collectionName}().To{collectionName}();
-        var b = p3.To{collectionName}();
-        var c = ImmutableExtensions.To{collectionName}(ImmutableExtensions.To{collectionName}(p1));
-        var d = ImmutableExtensions.To{collectionName}(p3);
+        var a = [|p1.To{collectionName}().To{collectionName}()|];
+        var b = [|p3.To{collectionName}()|];
+        var c = [|{collectionName}.To{collectionName}({collectionName}.To{collectionName}(p1))|];
+        var d = [|{collectionName}.To{collectionName}(p3)|];
```

Comment:
📝 The test source for immutable collections had deviated from reality. These tests were updated to reflect the new APIs.